### PR TITLE
SVG Icons for pumps fragment tab

### PR DESCRIPTION
**Pull request by Rig22**
I built some svg pump icons and put them in the appropriate layouts.

danars
pod
veo
combo
insight

### CODE REVIEW

**Comment by CaroGo** [comment]
Would love to see them integrated!

**Comment by swissalpine** [comment]
Sorry for my question: What's the benefit of these icons?

**Comment by Rig22** [comment]
Benefit, absolute nothing. Onlyo looking nice. 😉

**Comment by osodebailar** [comment]
Very nice ,
WhatsApp about the optical Aspect to Place the svgs to the Top of the Page , so as i did in my gui Fork ?

**Comment by Rig22** [comment]
Thank you. 🙂
I think both approaches have their advantages and disadvantages. I found that the graphic below gives the actual pump information more space without having to scroll if necessary.

**Comment by swissalpine** [comment]
@Rig22 Thank you. Ok, something pretty.
Please don't move it to the top of the tab. It is as you've written: For me it would be annoying to scroll down just for looking at a pretty but useless graphic.

**Comment by MilosKozak** [comment]
it would be nice to see then in ConfigBuilder in pump selection

File: core/src/main/res/drawable/ic_danars.xml [hunk 1]
```diff
@@ -0,0 +1,64 @@
+<vector android:height="100dp" android:viewportHeight="300"
```
Comment (MilosKozak) [inline core/src/main/res/drawable/ic_danars.xml:1]: should be placed under dana module

Comment (Rig22) [reply core/src/main/res/drawable/ic_danars.xml:1]: Hello Milos. :) Thanks for your review.

I'm not sure I get it right. Should the graphic be moved to AndroidAPS / dana / src / main / res /? There is no drawable folder there. Should I create one? Sorry, I'm not a programmer.....

**Comment by Rig22** [comment]
> it would be nice to see them in ConfigBuilder in pump selection. they would be usefull there

Thank you. :) 

Great idea with the Configbuilder. I'll have a look at it tomorrow. Let's see if I can do that?

**Comment by Rig22** [comment]
@MilosKozak 

I looked at the Config Builder thing. But that's too difficult for my very limited programming knowledge. If others like it, someone else would have to build it in.

**Comment by osodebailar** [comment]
graphic in bottom or top is not so important here. for me in top is the better idea ( in my gui fork i have the graphis in top because on the right side i integrate the bottom menu bottoms as the newer material fab buttons ).
So i am free where to place the icons in the old gui.

@MilosKozak 
the idea of placing icons in the configbuilder is a good idea.
i played with it a While and my suggestion would be :
extend the plugin datastructure with an icon resource for each plugin. The additional advantage is we can place the icons on the left side of the navigation menue in the navbar.
But i would to seperate this things.
A little PR for the icons on the pump side and than, because there are more things to do, a additional pr with the icon extension for the config builder.

@Rig22 if @MilosKozak  wants it, than i can help and do it.

**Comment by osodebailar** [comment]
@MilosKozak   @Rig22 the solution can be look like this example.
Its created on my gui fork, but the look and feel here is the same as in the original gui.

@Rig22 the image size is fixed to 36x36dp , may be you have to create seperate icon for it for a better look and feel . the omnipod icon has to expand here

Screenshot_1607350931

**Comment by Rig22** [comment]
> @Rig22 if @MilosKozak wants it, than i can help and do it.

Hi Andreas. :) Would be great. I added you as a collaborator...

**Pull request merged**